srcpkgs/linux-firmware/template

```diff
@@ -94,3 +95,11 @@ linux-firmware-network_package() {
 		vmove usr/lib/firmware/qca
 	}
 }
+
+linux-firmware-qualcomm_package() {
+	short_desc+=" - Qualcomm SoC blobs"
+	nostrip=yes
+	pkg_install() {
+		vmove usr/lib/firmware/qcom
+	}
+}
```

Comment:
I think network should depend on these?

Reply:
Technically yes as it contains the modem remoteprocessor firmware for at least SDM845 (which is obligatory for WLAN), but otherwise no; `qca` is the one containing Qualcomm Atheros FW and is already a part of that subpackage.

Reply:
> Technically yes as it contains the modem remoteprocessor firmware for at least SDM845 (which is obligatory for WLAN), but otherwise no

That would still make it a final "yes", wouldn't it?

Reply:
Let's compromise: would you accept adding `-qualcomm` as a dep for `-network` under `aarch64*|armv*` only? these blobs are only meant to be run under ARM platforms so it doesn't make sense to bring them in on e.g. x86 platforms.

Reply:
That sounds reasonable. Why not `arm*` instead of only v7?

I hope the extra separation doesn't come to bite us :p

Reply:
According to [the Qualcomm SoC wikipedia list](https://en.wikipedia.org/wiki/List_of_Qualcomm_Snapdragon_processors) there are only a few `armv6` SoCs from circa 2008 (which don't have anything in `linux-firmware/qcom` and I doubt the situation will change considering how low performance these chips are these days) and the rest are `armv7` or `aarch64`.

Reply:
Ok. Since a lot of consideration went into the split, please explain why things are being done this way in the commit message (basically just repeat what you wrote in previous comments)

Reply:
Done, do tell me if I missed something important :)

Reply:
Great, thanks :)